core/config/project_settings.cpp

```diff
@@ -1138,7 +1138,7 @@ Array ProjectSettings::get_global_class_list() {
 	Ref<ConfigFile> cf;
 	cf.instantiate();
 	if (cf->load(get_global_class_list_path()) == OK) {
-		script_classes = cf->get_value("", "list");
+		script_classes = cf->get_value("", "list", Array());
```

Comment:
This shouldn't be needed anymore since the other change makes sure to generate the file, but for good measure... :)